### ISSUE TITLE
Make sure b2b checkout works with keyboarding

### ISSUE THREAD

**Issue by LTiger14** [SAP/spartacus#8687]
A user should be able to checkout using the keyboard

**Comment by LTiger14** [comment]
Works flawlessly.
